sites/pillar.py

```diff
@@ -0,0 +1,26 @@
9evj61mYuRLDOWR/ke0pD7NW0iNykPlQf5v/t+qOIJ9lk7y0/xz27ZxevVNl92nvIrU1KgaFaKeD5Kl8jRMrJHKEonvciUN0Dzx5CHejRdmqIRc5bUP5fWs6pUiKMZy8v+pBWX+BXDytTkLhr0hAFHF9TjNq0ghIFGaL2ZVdru3SMxMk5fj16Cy8qQ0usqriUkL7UnrvP1bkc+2K8Vf9ymT9Xz2aVKO8oC6s5usVFjDCLRsR5pKY/RaXktvK3Dc3Uhu2mkxsU5RrAu1QglThS/OEXBX1aJhgA3YLbsWYSCvdB4rW7WIxo0pfv20lF4BbwM1XNpujkX5EeyIeN7cm0URfVuxXvKlQEXVszTEO0nCf1pIskC+gDZbb/CByrcUzFhEoN1HLMkHSDH6DHkZcyxO0XaX0iz8Zyxd8EsjzxTgeWksZdQLlvjXnYHVvoowANZGjRdP5cozKE2TdOQl/E+LCDNQlQxZxGMcYg3L63LqTjq0sLxLWrEqVj9Ro07jVI4CATFt3FWKAk/bpzGMonr3hHzqI8CJLNUj5/Z8GBJCynVjbCs6f2hZYpLv8M4hLQvk+pJFhJ45g/JXxm2+ZRcbJCeUJQhNJRGnU5N/Mrl9MM0HM5ms1o3NSWmTRSZ7YyQT3g59La1vddo+QUC+eQhVP+Uq5km+JzKat+mGMj3bfo2pH8oF3ygSvapuJqme0+YhIoqtczbrmn9KXvJ4GlaB10fyAI03UiDiQWL/tHWOhGLjKMzqT0r+odZvLo64C11/o0QipBBoXsIunjxtHKUg3zZNw/1So7t+wSUIl/sFZdzJQFNsU7KVvrqQ9w053LZpxsJ/MCfqOOI5O2LQ3cRcLpdJ4+4LtszvqO25uvRE42WNJ84hI1b5poGQF9rqEyuX1nxF0qeVhO5sgOob23KpStyTzh9s00DjQcO5PYYtuTTglly0Nl/mdhshVT13hXMEc6MOA6slx7cEADQyOoFtHTJeW2yZqfW41KzLGdgQXWSe/ogO0qcBbAQmgCYXMIyVDV0OtWuOZxNtayZJP4KWdMl2+g1AtMyfTIINtjNZrCj9KUVmQQJV/uMNhqBRdxyQg9LPauCqvezH52wXQNDT/JHRtm2pa0xX/cZ2pRrEm4m879RGNokOtWGscQGiMpPkTmRl00q/viVsSqeeEiyvOkvpLx81A7b6ZiQWSlGD60PTX26XDAlXjG7AWYPEjkxnNO5O3bIXXNfojmPPv0M2bkTkAOrLa/rFA5n9g22X3CfWDhXfSr9Tr82rR1iOOJ0jUP6RWFiZXWWck8gl0iYQOQ5Of7lZOKx3olin1YAKkW2Hd/0kcdbr9SLxg7ut1SNtEztJpOy0OZIN/sPtWTHW8K6JUK30p7jWUImNpDpigX6iZHFXYc+hPR0qftDB6afynldwgbDarnE6yQRzEgI6zNk58tGrPRCNtFt5LzvdmyvsP6d+91E4Gtg1DbCNfneI6xF/3JPfwVL56MZoTrD87rom0tzOyCaKxjw8/dQyyONqwl7f+jgJG7p9JyACqbx5QUdfKYvY8+rGAyCAhQHjlhltqVS6jf7xBkdVG/dCznSeYy3x/dvlEKje/mhoU5XlRlhZ4PEI9LN1caF2Vz2aG/uWvV8HINJfKwrS9dVtlO39ikf3gQmKVP0oLctExFvvoH9TfXBDNFfov6CY2ThShpMJsse5JW4jeRfRY9DfmQ4LVQGG9cDkGn7Y27l4RSXVZJY2QqBuX/XGZodjP6c/p2qa/feffwg5F3XJDvrrObVUmmX6EVysKJKIaJKJP74jm9E4Hi80dCcNNEehv+OW6V2gCYte6643t20EtNa3IN/DlMy0oDlRYqQYnBqcPXPK/KPwISwrBFxdH5K+WDLURn+X0l8xbvQU5cMMz44xQFIiInfXi31XoDBnMcdx6M/lomCZaNLd9EU/LNfJh5DP6MsndRdDcOa3+N96oNZEUKP2kE+60PCAd/ke9HfmuE5jd3usXpyhsnw6E6lwf7okfP8bKNueR6K/Y2RlXgsSVaBjZzkfbKKQzyOwwmgz6AeLtsio7g3V2a9gz0kD+4IKSmVF7UP/BNcmPRUNibya/kQTAwwjQpiB2p1jaaejY9FPe5wvl7JtMuvGudlfwlmMdkSd3PgWqoqH+sKHirMxOARnLEa2D/1UQ0XMMBshWfXX9HczcWqD69tcCegTIpkNAXnmcr97w+D2CoCpasvshTgfwRT2wXb/8PW+CBCJklEcx+uQnMv0214Jcd83L7LDZDQexwuLDGX7kur+Ud56cQdsof5lHJGqDNWLQCiQQek3R+UtDixVHdfh0XuiLRdPb94mQR85WdNXAP0jJBf1yZzMiBWGM1uTDE+zRy4K/PdOPEzd+BnYzsMLosRFOuc2+8IsNGsDR59c1LvOGMvehSDvQY/0im/8gw9NP6WwSpoTa9kMdu+G4SXbZjACEvM42a14ddu/u7tbTuQ64EXA9GOwll656dpPafNgrT6Nw9r7K0/oL90Jklrfe1Td1ufW/Bb+MYaArtnj690ieO7vqYPGk3SDmkVBWFv9PzpsLxQF26rR61Zo7OFYLD3TuErVvnFYtL/F07q74zuHgbuGXH9DDKO5Ikct6DrLgdWMgGpa79FpMie0w+jPOJcQYC3cpJOdSSp3HWSR1VaPCLH9vJz9CQz6IkJVYLPY+wtrd4GjxBbidZJGWNkXrDZ48N2BEov/QhhBBIXIcr5ULLF7swuwHZaILi56NkGgWXde2jtadiol/BbE5rCRR9f4Ln1tZwRgbX7SO9+K0e2gvcL5cUBizZNnugwntGGUzLXGlKLHuccoepa7cMJjiFdmRNC+aS9AcOZ7J7VzQBjxKrXsp3uAjs2zgem1FECf8E3QjfHiIZzRLtjVBxATm2xNr23K8QnfDt01xpM06lFbE0tmPgSI2DM7Mxdd4yT3R4Wu6/HKT+f15rMwzLIgGHUfK0Y8oYn/AbWixczpQB2YAAAAAElFTkSuQmCC"
```

Comment:
Couldn't find any images except in base64? It's not a problem, but it's hard to read.